Src/VimWpf/RelativeLineNumbers/ILineFormatTracker.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Windows.Media;
+
+namespace Vim.UI.Wpf.RelativeLineNumbers
+{
+    public interface ILineFormatTracker
+    {
+        Brush Background { get; }
+
+        System.Windows.Media.TextFormatting.TextLine MakeTextLine(int number);
+
+        double NumberWidth { get; }
+
+        bool Numbers { get; }

```

Comment:
Can you add a comment to what this property does? It's not clear at a glance.

Reply:
Sure. Actually it should be "Number" and "RelativeNumber" or "VimNumber" and "VimRelativeNumber" as they are properties for Vim options.